jaxopt/_src/eq_qp.py

```diff
@@ -31,9 +31,10 @@
 from jaxopt._src.linear_operator import _make_linear_operator
 
 
-def _check_params(params_obj, params_eq):
+def _check_params(params_obj, params_eq=None, params_ineq=None):
```

Comment:
Strictly speaking, it's a bit strange to have `params_ineq` in a file called `eq_qp.py`... Maybe do it the other way around: move `_check_params` to `cvxpy_wrapper.py` and import it here...